core/backend/src/core-backend.ts

```diff
@@ -15,6 +15,7 @@ export * from "./ChannelControl";
 export * from "./CheckpointManager";
 export * from "./ClassRegistry";
 export * from "./CloudSqlite";
+export * from "./CloudSqliteMock";
```

Comment:
do we need to export this from the main barrel?
should this internal api be imported from some sub path?

Reply:
I copied what is being done with `HubMock`, which is also marked as internal. Both classes are imported and used in full-stack-tests and example-code.

Reply:
I'll hide CloudSqliteMock and do its startup and shutdown inside the startup and shutdown for HubMock.

Reply:
With startup/shutdown happening inside HubMock, I am no longer exporting CloudSqliteMock.